File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/BodyStructure.ts
import { Attachment } from "./Attachment";
import { CodeableConcept } from "./CodeableConcept";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type BodyStructure = DomainResource & {
    image?: Attachment[];
    active?: boolean;
    patient: Reference;
    location?: CodeableConcept;
    identifier?: Identifier[];
    morphology?: CodeableConcept;
    description?: string;
    locationQualifier?: CodeableConcept[];
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/BodyStructure.py
from typing import Optional, List
from base import Attachment
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

class BodyStructure(DomainResource):
    image: Optional[List[Attachment]] = None
    active: Optional[bool] = None
    patient: Reference
    location: Optional[CodeableConcept] = None
    identifier: Optional[List[Identifier]] = None
    morphology: Optional[CodeableConcept] = None
    description: Optional[str] = None
    location_qualifier: Optional[List[CodeableConcept]] = None